src/vs/workbench/workbench.common.main.ts

```diff
@@ -479,4 +479,7 @@ import 'sql/workbench/contrib/extensions/browser/extensions.contribution';
 // Azure
 import 'sql/workbench/contrib/azure/browser/azure.contribution';
 
+// Onboarding
+import 'sql/workbench/contrib/onboarding/common/onboarding.contribution';
```

Comment:
any way or need to avoid vs change?

Reply:
I think we could potentially add our own workbench file and load that alongside this, but this is essentially no risk and I think thats why we've never done that.

Reply:
I would suggest having just an extension command contributed via the extension rather than adding a contribution in core, but this can be open to discussion

Reply:
I think it makes more sense to make this a extension that doesn't ship.

Reply:
What if we just put it in integration tests? I don't think doing the entire infrastructure to add another extension for this makes sense at all.

Reply:
I mean, the integration tests are an extension that doesn't ship so that meets what I was saying.